### PR TITLE
Cleanup dependencies

### PR DESCRIPTION
**Pull request by Eijebong**
This change is [Reviewable](https://reviewable.io/reviews/servo/rust-azure/281)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -11,18 +11,14 @@ build = "build.rs"
...
 
 [dependencies]
 euclid = "0.15"
-heapsize = {version = "0.4", optional = true}
-heapsize_derive = {version = "0.1.0", optional = true}
 libc = "0.2"
-serde = {version = "1.0", optional = true}
-serde_derive = {version = "1.0", optional = true}
+serde = {version = "1.0", features=["derive"], optional = true}
 servo-skia = "0.30000008"
```
Comment (SimonSapin) [inline Cargo.toml]: r+ with 0.30000009 here (yet to be published).

Comment (Eijebong) [reply Cargo.toml]: Well it's not bumping euclid either. I can include it in this PR if you want to

Comment (SimonSapin) [reply Cargo.toml]: Oh right, sorry. Renaming the feature is a breaking change and the skia update is close, so maybe let’s land everything together?

Comment (Eijebong) [reply Cargo.toml]: Sure

**Comment by SimonSapin** [comment]
@bors-servo r+

Thanks!

**Pull request merged**